cmd/restic/cmd_copy.go

```diff
@@ -252,5 +380,54 @@ func copyTree(ctx context.Context, srcRepo restic.Repository, dstRepo restic.Rep
 	if err != nil {
 		return errors.Fatal(err.Error())
 	}
+
+	// create all the snapshots
+	for _, sn := range selectedSnapshots {
+		// save snapshot
+		sn.Parent = nil // Parent does not have relevance in the new repo.
+		// Use Original as a persistent snapshot ID
+		if sn.Original == nil {
+			sn.Original = sn.ID()
+		}
+		newID, err := restic.SaveSnapshot(ctx, dstRepo, sn)
+		if err != nil {
+			return err
+		}
+		Verbosef("snapshot %s saved\n", newID.Str())
+	}
+
 	return nil
 }
+
+// printStats: generate short statistic for blob / packs / snapshots to be copied
```

Comment:
Remove the stats function here and let's discuss it in https://github.com/restic/restic/pull/5464 .